plugins/webkul/projects/src/Filament/Clusters/Configurations/Resources/ProjectStageResource.php

```diff
@@ -83,6 +83,7 @@ public static function table(Table $table): Table
                             ->body(__('projects::filament/clusters/configurations/resources/project-stage.table.actions.restore.notification.body')),
                     ),
                 Tables\Actions\DeleteAction::make()
+                    ->hidden(fn ($record) => $record->projects->isNotEmpty())
```

Comment:
We shouldn't hide the delete option instead we should handle the archive categories